vmslave-config/pipeline/group_vars/deb

```diff
@@ -1,2 +1,2 @@
 # on-imagebuilder is not working in ansible more than 2.0
-slave_ansible_version: "{{ lookup('env','SLAVE_ANSIBLE_VERSION') }}"
\ No newline at end of file
+ansible_version: "{{ lookup('env','DEB_ANSIBLE_VERSION') }}"
```

Comment:
The (re)naming here is to make the var name "ansible_version" have the same format with other vars.  but I forgot why I name this var to "slave_ansible_version". 

```ansible_version``` is a default system var of ansible. Its value is the version of ansible in master.

So now I changed the var name to ```slave_ansible_version``` as before